mobilevit/models/transformer_block.py

```diff
@@ -0,0 +1,58 @@
+import tensorflow as tf
+from tensorflow.keras import layers
+
+
+# Reference:
+# https://keras.io/examples/vision/image_classification_with_vision_transformer/
+def mlp(input_layer, hidden_units: int, dropout_rate: int):
+    """
+    MLP layer.
+
+    Args:
+        input_layer: input tensor.
+        hidden_units (int): list of hidden units.
+        dropout_rate (int): dropout rate.
+
+    Returns:
+        output tensor.
+    """
+    for units in hidden_units:
+        x = layers.Dense(units, activation=tf.nn.swish)(input_layer)
```

Comment:
The usual choice for the activation here is GeLU. Since it deviates from that, add a note reflecting the change.